client/src/app-components/ticket-list.js

```diff
@@ -49,25 +50,28 @@ class TicketList extends React.Component {
     };
```

Comment:
We have a new prop, so let's add its prop type: `showDropDown` (change its name too, see other comments).